packages/react/src/table/docs/overview.mdx

```diff
@@ -91,21 +91,25 @@ For data tables with a small amount of rows, use the default table. Align text c
 - use for datasets with more than 2 columns
 - use for tabular data - not layouts
 - provide a meaningful caption (heading), or use aria-labelledby to reference a heading
-- keep row heights small.
+- keep row heights small
 - use a simple table structure - complex tables are difficult to navigate.
 - use Pagination to break up tables with many rows
 - align text left and numeric data right
-- only include text, tags, links and status badges as content
-- use a human readable format for numeric data – for example, 1,234,000 instead of 12340000.
+- only include text, tags, links, actions and status badges as content
+- only use dropdown menu for actions if there are three or more essential actions
```

Comment:
"important" sounds better than "essential" imo

```suggestion
- only use dropdown menu for actions if there are three or more important actions
```

Reply:
I think it needs to be 'required' because actions on tables should only be used if required

Reply:
"Required" sort of makes it sound like a user has to perform these actions on the table rows

Reply:
I think we remove the adjective all together and have
```suggestion
- only use dropdown menu if you require three or more actions in a table cell
```

Reply:
Yeah, that's much better